### PR TITLE
Global lb

### PR DESCRIPTION
**Pull request by yuna-iwata**
- global leaderboard displayed in front end. fetch to api done in networking

### CODE REVIEW

File: src/Components/Leaderboard.jsx [hunk 1]
```diff
@@ -7,29 +7,25 @@ import TableRow from "@mui/material/TableRow";
...
+import { useState } from "react";
+
+export default function Leaderboard({ baseUrl, itemData }) {
+  const [globalList, setGlobalList] = useState([]);
+
+  useEffect(() => {
+    fetchGlobalLeaderBoard(setGlobalList);
+  }, []);
+
+  useEffect(() => {
```
Comment (louisglan) [inline src/Components/Leaderboard.jsx]: love this

File: src/Components/Leaderboard.jsx [hunk 2]
```diff
@@ -7,29 +7,25 @@ import TableRow from "@mui/material/TableRow";
...
-              <CardHeader
-                avatar={<Avatar alt="Dino profile" src="/dino-idle.png" />}
-                title="Yuna"
-              />
-            </TableCell>
-            <TableCell align="center">9999</TableCell>
-          </TableRow>
-          {/* 
-          {rowList.map((row) => (
+          {globalList.map((row) => (
```
Comment (louisglan) [inline src/Components/Leaderboard.jsx:24]: love it

**Review by louisglan — CHANGES_REQUESTED** [review]
just the useEffect console.log that needs removing

File: src/Components/Leaderboard.jsx (hunk 2 above)
Comment (yuna-iwata) [reply src/Components/Leaderboard.jsx:24]: ❤️

File: src/Components/Leaderboard.jsx (hunk 1 above)
Comment (yuna-iwata) [reply src/Components/Leaderboard.jsx]: thank u i try

**Pull request merged**